2019-06-18 17:31:38,207 ERROR: Exception on /recipe/Молочный коктейль [POST] [in c:\python\git\microblog\winvenv\lib\site-packages\flask\app.py:1780]
Traceback (most recent call last):
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask\app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask\app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask\app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask\_compat.py", line 36, in reraise
    raise value
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask\app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask\app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\python\git\microblog\winvenv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Python\git\microblog\app\routes.py", line 246, in recipe
    ingredient = Ingredient.query.filter_by(name=form.ingredient.data).first()
AttributeError: 'IngredientForm' object has no attribute 'ingredient'
2019-06-18 17:32:33,877 INFO: Microblog startup [in C:\Python\git\microblog\app\__init__.py:54]
2019-06-18 17:34:32,761 INFO: Microblog startup [in C:\Python\git\microblog\app\__init__.py:54]
2019-06-18 17:54:08,191 INFO: Microblog startup [in C:\Python\git\microblog\app\__init__.py:54]
